libs/designsystem/checkbox/src/checkbox.component.spec.ts

```diff
@@ -19,7 +19,7 @@ const checkboxSizeMd = size('xxxl');
 describe('CheckboxComponent', () => {
   const createComponent = createComponentFactory({
     component: CheckboxComponent,
-    imports: [IonicModule.forRoot({ mode: 'ios', _testing: true })],
+    providers: [provideIonicAngular({ mode: 'ios', _testing: true })],
```

Comment:
Consider using the testhelper here
```suggestion
    providers: [TestHelper.ionicModuleForTest],
```